pandas/core/common.py

```diff
@@ -517,7 +517,7 @@ def f(x):
 
 
 def convert_to_list_like(
-    values: Scalar | Iterable | AnyArrayLike,
+    values: Scalar | Iterable | AnyArrayLike | Hashable,
```

Comment:
Could probably replace `Scalar` with `Hashable`, all scalars should also be `Hashable`

Reply:
fixed in next commit